lib/SILOptimizer/Utils/ConstantFolding.cpp

```diff
@@ -502,6 +502,44 @@ static SILValue constantFoldCompare(BuiltinInst *BI, BuiltinValueKind ID) {
   return nullptr;
 }
 
+/// Determines if a FunDecl has the attribute: @_semantics("ignore.overflow").
+bool hasIgnoreOverflowAttr(FuncDecl &fDecl) {
+  for (auto *semAttr : fDecl.getAttrs().getAttributes<SemanticsAttr>()) {
+    if (semAttr->Value == "ignore.overflow")
```

Comment:
Nit: looks like lately the naming of these attributes have changed from snake case to [camel case](https://github.com/apple/swift/blob/88d305caded65d4401ee06b4dd4f8c0d621e1c34/stdlib/public/core/StringSwitch.swift#L70), with dots reserved for things like “array.init”—maybe this could be “integer.ignoreOverflow”?

Reply:
I see. I will make this change.